routes/mergeDataConflictGuide.js

```diff
@@ -3,7 +3,10 @@ var linz = require('../');
 /* GET /admin/merge-data-conflict-guide */
 var route = function (req, res) {
 
-    res.render(linz.api.views.viewPath('merge-data-conflict-guide.jade'), {});
+    res.render(linz.api.views.viewPath('merge-data-conflict-guide.jade'), {
+        scripts: res.locals.scripts,
```

Comment:
I'm not really sure, the docs don't seem that clear to me http://expressjs.com/en/api.html#res.locals